src/main/rx/rx.c

```diff
@@ -84,6 +92,28 @@ void useRxConfig(rxConfig_t *rxConfigToUse)
 }
 
 #define STICK_CHANNEL_COUNT 4
+#define REQUIRED_CHANNEL_MASK 0x0F // first 4 channels
+
+// pulse duration is in micro seconds (usec)
+void rxCheckPulse(uint8_t channel, uint16_t pulseDuration)
+{
+    static uint8_t goodChannelMask = 0;
+
+    if (channel < 4 &&
```

Comment:
In #185 i supplied the number of channels that must be checked in this function call.